doc/rose-command.html

```diff
@@ -954,6 +959,11 @@ <h2 id="rose-date">rose date</h2>
     <h3>OPTIONS</h3>
 
     <dl>
+      <dt><kbd>--as-total=TIME_FORMAT</kbd><dt>
```

Comment:
The `t/syntax/01-doc.t` rose test-battery test notices that this should be a `</dt>` - I didn't!